src/__generated__/AllArtistsFollowArtistMutation.graphql.ts

```diff
@@ -0,0 +1,118 @@
+/* tslint:disable */
+
+import { ConcreteRequest } from "relay-runtime";
```

Comment:
@kierangillen Looks like the sorting is happening again.. we should find a way to exclude those or what's causing that on your machine